### PR TITLE
Crier gerrit reporter: ensure reports all jobs

### PR DESCRIPTION
**Pull request by chaodaiG**
Gerrit adaptor depends on comments posted by crier gerrit reporter to figure out which prow jobs to be triggered with /retest. #24770 truncates gerrit reports when there are too many prow jobs, resulted in not all prow jobs are reported in the comments. So when a user comments /retest prow only triggers the jobs that were included in the report, instead of all prow jobs that failed.

### CODE REVIEW

**Comment by chaodaiG** [comment]
cc @2nd-half

**Comment by chaodaiG** [comment]
This PR changes from a single gerrit comment to 2 comments:
- One vote on gerrit labels, comment doesn't contain URL, e.g. `jobA succeed`
- The other is pure comment with URL, e.g. `jobA succeed - https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/batch/pull-kubernetes-unit/1478569178959974400`

The first comment is short and helps ensure that all jobs are reported in the comment, which can then be used by gerrit adaptor to decide which jobs are failed. The second comment is helpful for helping debugging.

Theoretically these two comments could be combined into a single comment, which would work for most cases when there are not a lot of presubmit tests. However for repos that contain 10 - 20 presubmit tests, adding more contents(the lines without URLs) might throw their comments over the size limit. On the other hand it probably won't look good

**Comment by chaodaiG** [comment]
/hold
This PR is based on top of #24770, the comment size limit was hard coded as 4000, which is actually configurable for each gerrit host. Will need to put more thoughts on it before proceeding

File: prow/crier/reporters/gerrit/reporter.go [hunk 1]
```diff
@@ -336,19 +338,26 @@ func (c *Client) Report(ctx context.Context, logger *logrus.Entry, pj *v1.ProwJo
 
 	logger.Infof("Reporting to instance %s on id %s with message %s", gerritInstance, gerritID, message)
 	if err := c.gc.SetReview(gerritInstance, gerritID, gerritRevision, message, reviewLabels); err != nil {
-		logger.WithError(err).Infof("fail to set review with label %q on change ID %s", reportLabel, gerritID)
+		logger.WithError(err).WithField("gerrit_id", gerritID).WithField("label", reportLabel).Info("Fail to set review.")
```
Comment (fejta) [inline prow/crier/reporters/gerrit/reporter.go:341]: "Failed to set review" sounds better than "Fail" (the failure already happened)

**Comment by chaodaiG** [comment]
close in favor of https://github.com/kubernetes/test-infra/pull/24789

/close

**Pull request closed without merging**